src/ui/components/SerialConnectionForm/index.tsx

```diff
@@ -123,7 +123,7 @@ const SerialConnectionForm: FunctionComponent<SerialConnectionFormProps> = (
   }, [data, previousData]);
   const onSubmit = () => {
     if (currentValue !== null) {
-      onConnect(currentValue?.value, baudRate);
+      onConnect(currentValue?.value, currentBaud);
```

Comment:
Can parse form value over here to make it a `number`

Reply:
Well, but the problem is currentBaud is already of type number. In the react setter we parse it:

```
        const value = parseInt(event.target.value, 10);
        setCurrentBaud(value);
```

Yet, for some reason it's sending a string across the wire. I don't really know GraphQL, so I'm not sure where it's getting converted to a string. Any suggestion is welcome.